calculator.js

```diff
@@ -0,0 +1,30 @@
+const calculator = (() => ({
+  add: (...args) => args.reduce((cr, acc) => {
+    // eslint-disable-next-line no-param-reassign
+    acc += cr;
+    return acc;
+  }, 0),
```

Comment:
Just return `acc + cr` as mentioned in `arrayAnalysis`.

Reply:
It was like that originaly but the linter complained about that :-(